front-end/src/components/Departments/Departments.jsx

```diff
@@ -0,0 +1,85 @@
+import './Departments.css';
+import Slider from "react-slick";
+import { useState, useEffect } from "react";
+import { Available } from '../Available';
+import { departmentsData } from "../../mocks/data";
+import { DEPARTMENT_TITLE } from '../../constants.js';    
+
+const departments = departmentsData.departments;
+
+function PrevArrow(props) {
+    const { className, style, onClick } = props;
+    return (
+      <div
+        className={className}
+        style={{
+          ...style,
+          display: "block",
+          height: "auto",
+          zIndex: 1000, 
+          position: "absolute", 
+          top: "12vw",
+          left: "1.5vw",
```

Comment:
should move css to css file and use arrow function to make it consistant with other function components

Reply:
i've created a SliderArrow component to be used in the Partners and Departments components